palace/drivers/eigensolver.cpp

```diff
@@ -148,13 +148,13 @@ EigenSolver::Solve(const std::vector<std::unique_ptr<Mesh>> &mesh) const
   std::unique_ptr<Operator> KM;
   if (iodata.solver.eigenmode.mass_orthog)
   {
-    // Mpi::Print(" Basis uses M-inner product\n");
-    // KM = spaceop.GetInnerProductMatrix(0.0, 1.0, nullptr, M.get());
-    // eigen->SetBMat(*KM);
-
-    Mpi::Print(" Basis uses (K + M)-inner product\n");
-    KM = spaceop.GetInnerProductMatrix(1.0, 1.0, K.get(), M.get());
+    Mpi::Print(" Basis uses M-inner product\n");
+    KM = spaceop.GetInnerProductMatrix(0.0, 1.0, nullptr, M.get());
     eigen->SetBMat(*KM);
```

Comment:
Worth calling out this change in default in the log?

Reply:
Done in [4ec2ca6](https://github.com/awslabs/palace/pull/168/commits/4ec2ca63727f6797a576803aafa0900c91771eeb).